.gitignore

```diff
@@ -0,0 +1,43 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
Was this autogenerated? Either way looks fine (the url doesn't seem to direct to an actual page and redirects to support)